Detectors/TPC/calibration/SpacePoints/src/TrackInterpolation.cxx

```diff
@@ -437,7 +437,7 @@ bool TrackInterpolation::validateTrack(const TrackData& trk, TrackParams& params
     LOG(debug) << "Skipping track too far from helix approximation";
     return false;
   }
-  if (fabsf(params.qpt) > mParams->maxQ2Pt) {
+  if (fabsf(mBz) > 0.01 && fabsf(params.qpt) > mParams->maxQ2Pt) {
```

Comment:
What is the reason of this protection? For B=0 the track is supposed to have default pT~0.6 GeV (modulo energy loss corrections). Asking just to be sure that you have not seen too large momentum tracks at B=0.

Reply:
This is just because the `params.qpt` here is recalculated within the `compareToHelix` method. And that does not work for B=0. So I have to avoid this check